values.yaml

```
@@ -1165,6 +1165,27 @@ nginx:
     setHeaders: {}
     # -- (optional) List of [auth tenants](https://cortexmetrics.io/docs/guides/auth/) to set in the nginx config
     auth_orgs: []
+    # -- (optional) Name of basic auth secret.
+    # In order to use this option, a secret with htpasswd formatted contents at
+    # the key ".htpasswd" must exist. For example:
+    #
+    #   apiVersion: v1
+    #   kind: Secret
+    #   metadata:
+    #     name: my-secret
+    #     namespace: <same as cortex installation>
+    #   stringData:
+    #     .htpasswd: |
+    #       user1:$apr1$/woC1jnP$KAh0SsVn5qeSMjTtn0E9Q0
+    #       user2:$apr1$QdR8fNLT$vbCEEzDj7LyqCMyNpSoBh/
+    #
+    # Please note that the use of basic auth will not identify organizations
+    # the way X-Scope-OrgID does. Thus, the use of basic auth alone will not
+    # prevent one tenant from viewing the metrics of another. To ensure tenants
+    # are scoped appropriately, explicitly pass the `X-Scope-OrgID` header
+    # along with a basic auth header.
```

This is worded backwards... _passing_ the header requires trusting the client. Setting the header in the nginx config with appropriately scope the tenants.
```suggestion
    # prevent one tenant from viewing the metrics of another. To ensure tenants
    # are scoped appropriately, explicitly set the `X-Scope-OrgID` header
    # in the nginx config. Example
    #   setHeaders:
    #     X-Scope-Org-Id: $remote_user
```

---

Ok. Updated.